AsyncDisplayKit/Details/ASBatchContext.h

```diff
@@ -34,6 +34,14 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)completeBatchFetching:(BOOL)didComplete;
 
 /**
+ * Let the context object know that a batch fetch was completed.
+ *
+ * @discussion For instance, when a table has reached the end of its data, a batch fetch will be attempted unless the context
+ * object thinks that it is still fetching.
+ */
+- (void)completeBatchFetching;
```

Comment:
Is this simply a convenience for the method above? If so, the discussion should probably just mention that it is equivalent to the above call passing YES, and there should be a @see annotation for the other method.